readme.md

```diff
@@ -302,23 +312,134 @@ Sets `options.method` to the method name and makes a request.
 
 #### Instances
 
-#### got.create([options])
+#### got.extend([options])
 
-Configure a new `got` instance with default `options`:
+Configure a new `got` instance with default `options` and custom `endpoint` (optional):
 
 ```js
 (async () => {
-	const client = got.create({headers: {'x-foo': 'bar'}});
-	const {headers} = await client.get('httpbin.org/headers', {json: true}).body;
+	const client = got.extend({headers: {'x-foo': 'bar'}, endpoint: 'httpbin.org/'});
+	const {headers} = (await client.get('headers', {json: true})).body;
 	//=> headers['x-foo'] === 'bar'
 
-	const jsonClient = client.create({json: true, headers: {'x-baz': 'qux'}});
-	const {headers: headers2} = await jsonClient.get('httpbin.org/headers').body;
+	const jsonClient = client.extend({json: true, headers: {'x-baz': 'qux'}});
+	const {headers: headers2} = (await jsonClient.get('headers')).body;
 	//=> headers2['x-foo'] === 'bar'
 	//=> headers2['x-baz'] === 'qux'
 })();
 ```
 
+#### got.create(settings)
+
+Example: [gh-got](https://github.com/sindresorhus/gh-got/blob/master/index.js)
+
+Configure a new `got` instance with provided settings:
```

Comment:
We should be more explicit here about `got.create()` having no good defaults included by default and how it's different from `got.extend()`.